Makefile

```diff
@@ -196,7 +205,7 @@ deploy-init:
 	$(eval export TF_VAR_paas_docker_image=ghcr.io/dfe-digital/apply-teacher-training:$(IMAGE_TAG))
 
 	az account set -s $(AZURE_SUBSCRIPTION) && az account show
-	terraform -chdir=terraform/$(PLATFORM) init -reconfigure -backend-config=./workspace_variables/$(APP_ENV)_backend.tfvars $(backend_key)
+	terraform -chdir=terraform/$(PLATFORM) init -reconfigure -upgrade -backend-config=./workspace_variables/$(APP_ENV)_backend.tfvars $(backend_key)
```

Comment:
Should we remove the `-upgrade`?

Reply:
We've pinned the provider versions exactly so there is no risk of upgrading to an unknown version